### DISCUSSION TITLE
I wonder what just happened in mania

### DISCUSSION THREAD

**Discussion by Sarisan** [ppy/osu#23846] [Q&A] [answered]
Was
![image](https://github.com/ppy/osu/assets/63128024/c9414b04-eace-49aa-80e8-6d0acce395b9)
Now
![image](https://github.com/ppy/osu/assets/63128024/772ffc87-67e8-4899-8abf-a3acbba2b3a9)

Recalculations after switching to ScoreV2?

**Comment by peppy** [comment, marked as answer]
The graph for today will update as today continues. This is normal.
